app/web/features/communities/CommunitiesPage/CommunitiesPage.tsx

```diff
@@ -0,0 +1,280 @@
+import {
+  Divider,
+  List,
+  ListItem,
+  ListItemText,
+  Typography,
+} from "@mui/material";
+import MuiLink from "@mui/material/Link";
+import Alert from "components/Alert";
+import Button from "components/Button";
+import CenteredSpinner from "components/CenteredSpinner/CenteredSpinner";
+import PageTitle from "components/PageTitle";
+import StyledLink from "components/StyledLink";
+import {
+  useCommunity,
+  useListSubCommunities,
+} from "features/communities/hooks";
+import { Trans, useTranslation } from "i18n";
+import { COMMUNITIES, DASHBOARD, GLOBAL } from "i18n/namespaces";
+import { Community } from "proto/communities_pb";
+import React, { useEffect, useRef, useState } from "react";
+import { routeToCommunity } from "routes";
+import makeStyles from "utils/makeStyles";
+
+const COMMUNITY_BUILDER_FORM_LINK =
+  "https://couchers.org/community-builder-form";
+const COMMUNITY_GUIDELINES_LINK =
+  "https://docs.google.com/document/d/1A9vCdKGQ_WKoyjCh5KScdGgpnighr9yL_F0-OaPT2yA/edit?usp=sharing";
+
+const useStyles = makeStyles((theme) => ({
+  root: {
+    display: "flex",
+    flexDirection: "row",
+    alignItems: "flex-start",
+    "& > * + *": {
+      marginInlineStart: theme.spacing(2),
+    },
+  },
+  list: {
+    minWidth: "10rem",
+  },
+  loader: {
+    margin: theme.spacing("auto", 2),
+  },
+  selected: {
+    fontWeight: "bold",
+  },
```

Comment:
Would you mind switching the styles for this to the new `styled` format? We're working on getting rid of `makeStyles` so we can upgrade the package so try to do new components the new way.

If you search `styled` or `Styled` you should see some examples! Just shout if you have questions about it I've done a billionnnnn conversions.